### PR TITLE
use conformance tag properly

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@stevekuznetsov @smarterclayton ptal.  most of these tests were already tagged conformance, a few weren't and i've added it, and there were 2 (noted in the file) which did not appear to match anything, so i'm looking for feedback on what you'd like done with those 2.

**Comment by smarterclayton** [comment]
[twst]

**Comment by bparees** [comment]
[test]

File: test/extended/setup.sh [hunk 1]
```diff
@@ -355,9 +355,7 @@ readonly CONFORMANCE_TESTS=(
 
 	"Services.*NodePort"
 	"ResourceQuota should"
-	"\[networking\] basic openshift networking"
-	"\[networking\]\[router\]"
-	"Ensure supplemental groups propagate to docker"
+	#"\[networking\] basic openshift networking" - NOT FOUND
```
Comment (bparees) [inline test/extended/setup.sh]: @smarterclayton so i'm reasonably convinced we can delete this one.

File: test/extended/setup.sh [hunk 2]
```diff
@@ -368,18 +366,13 @@ readonly CONFORMANCE_TESTS=(
...
-	"\[volumes\] Test local storage quota FSGroup"
-	"test deployment should run a deployment to completion"
 	"Variable Expansion"
 	"init containers"
 	"Clean up pods on node kubelet"
 	"\[Feature\:SecurityContext\]"
 	"should create a LimitRange with defaults"
 	"Generated release_1_2 clientset"
-	"\[Feature\:PodDisruptionbudget\]"
+	# "\[Feature\:PodDisruptionbudget\]" - does not match anything, perhaps intended to match tests in disruption.go ?
```
Comment (bparees) [inline test/extended/setup.sh]: @smarterclayton but i'm not sure what you'd like to do about this one.  I can change the focus to something that actually matches the tests in disruption.go, or i can delete it. your call.

**Comment by smarterclayton** [comment]
Remove it and in the future I'll go review adding it back in.

**Comment by bparees** [comment]
> Remove it and in the future I'll go review adding it back in.

@smarterclayton done.

**Comment by smarterclayton** [comment]
[merge

**Comment by smarterclayton** [comment]
[merge]

**Pull request merged**